app/src/utils/data/grants.ts

```diff
@@ -155,7 +155,12 @@ export function grantListener(name: string, refs: Record<string, Ref>) {
         grantId = BigNumber.from(grantId).toNumber();
 
         // update the grants metadata
-        if (_lsGrants[grantId] && _lsGrants[grantId].metaPtr !== metaPtr) {
+        const areMetaPtrsEqual = (x: MetaPtr, y: MetaPtr) => {
+          if (BigNumber.from(x.protocol).toString() !== BigNumber.from(y.protocol).toString()) return false;
+          if (x.protocol !== y.protocol) return false;
```

Comment:
^ worth combing the two if statements ?

Reply:
Heh, you could, but I did this because it takes up less lines without compromising readability which I like. But since @mk1r gave you the +1 I'm outvoted and will change this

Actually changing it to just `return BigNumber.from(x.protocol).toString() === BigNumber.from(y.protocol).toString() && x.protocol === y.protocol` seems the simplest

Reply:
Changed in cddb3fa